bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/Runtime.java

```diff
@@ -81,8 +81,8 @@ public Object invokeMethodAsync(BObject object, String methodName, String strand
          if (isIsolate) {
              return scheduler.schedule(new Object[1], func, null, callback, strandName, metadata).result;
          } else {
-             return scheduler.scheduleToObjectGroup(object, new Object[1], func, null, callback, strandName, metadata)
-                     .result;
+             return scheduler.scheduleToObjectGroup(object, new Object[1], func, null, callback, null,
```

Comment:
Shall we call the this invokeMethodAsync method with other function.

ex
      */
     public Object invokeMethodAsync(BObject object, String methodName, String strandName, StrandMetadata metadata,
                                     Callback callback, Object... args) {
         return invokeMethodAsync(object, methodName, strandName, metadata, callback, null,
                           PredefinedTypes.TYPE_NULL, args).getResult();

     }

Reply:
Maybe we can do as a separate PR to master. Then we do not need to duplicate  if (isIsolate) { logic .

Reply:
I think two functions are duplicates, shall we remove one of them altogether ?